lib/data_update_scripts/20201030134117_reindex_users_for_username_search.rb

```diff
@@ -0,0 +1,11 @@
+module DataUpdateScripts
+  class ReindexUsersForUsernameSearch
+    def run
+      Search::User.update_mappings
```

Comment:
No need to run this, this is run by our deploy processes so it is done well before any of the code hits production servers.

Reply:
Removed!